packages/web/src/components/MeterPointInfo/MeterPointInfo.tsx

```diff
@@ -16,113 +16,118 @@ import {
   getBuildingNumber,
   getCustomerNumber,
 } from '@utils/extractDataHelper'
+import { getUrlWithParams, routes } from '@routing'
 
 interface MeterpointInfoProps {
   contract: Contract
+  customerId: string
 }
 
-export const MeterPointInfo = ({ contract }: MeterpointInfoProps) => {
-  const { hanPortEvent, loading, contractById } = useSelector(
+interface TableRowData {
+  headerKey: string
+  headerContent: string | ReactElement
+  fieldKey: string
+  fieldContent: string | ReactElement
```

Comment:
```suggestion
  headerContent: React.ReactNode
  fieldKey: string
  fieldContent: React.ReactNode
```
This type includes everything that can be rendered by react and is the parent type for the ones you have used.

Reply:
Oh, thanks!